pipelines/build/dockerFiles/dragonwell.dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG image
+
+FROM $image
+
+RUN \
+    # Dragonewell 8 requires a dragonwell 8 BootJDK
```

Comment:
So we're only doing 8 for now?

Reply:
Dragonwell 11 can be built with a regular bootJDK. This is specific to 8